AdminCore.Services.Tests/DashboardServiceLogicTests.cs

```diff
@@ -18,10 +18,10 @@ public DashboardServiceLogicTests()
     [Fact]
     public void TestEmployeeDashboardEventsLogic_Check1ResultComesBackForEmployeeId1ForDecember()
     {
-      // Employee 1 has 2 December 2018 events. One approved and one cancelled. This query should only return the one that is approved.
+      // Employee 1 has 2 December 2018 events. One Awaiting approval and one cancelled. This query should only return the one that is awaiting approved.
       var resultList = _mockDatabase.EventRepository.Where(evnt => DashboardService.EmployeeDashboardEventsQuery(1, new DateTime(2018, 12, 5), evnt)).ToList();
       Assert.Single(resultList);
-      Assert.Equal((int)EventStatuses.Approved, resultList.First().EventStatusId);
+      Assert.Equal((int)EventStatuses.AwaitingApproval, resultList.First().EventStatusId);
```

Comment:
reason for this change is unclear, why was this changed?

Reply:
The eventRepo didn't contain an event that was Approved with these particular parameters, so the test was failing. So I changed the assertion to assert for an event that did exist in the mock db.